XenOvfApi/Utilities/ValueMaps.cs

```diff
@@ -181,17 +181,22 @@ private static void LoadMap(XmlDocument xd, string fieldname, Dictionary<string,
             XmlNodeList xnl = xd.SelectNodes("CIM/DECLARATION/DECLGROUP/VALUE.OBJECT/CLASS/PROPERTY");
             foreach (XmlNode xn in xnl)
             {
+                if (xn.Attributes == null) 
+                    continue;
                 foreach (XmlAttribute xa in xn.Attributes)
                 {
                     if (xa.Name.ToUpper().Equals("NAME") && xa.Value.ToUpper().Equals(fieldname.ToUpper()))
                     {
                         foreach (XmlNode xn1 in xn.ChildNodes)
                         {
-                            if (xn1.Name.ToUpper().Equals("QUALIFIER") && xn1.Attributes.GetNamedItem("NAME").Value.ToUpper().Equals("VALUEMAP"))
+                            if (xn1.Name.ToUpper().Equals("QUALIFIER") && xn1.Attributes.GetNamedItem("NAME").Value

```

Comment:
I think this would be more readable if the line break were after the `&&`. Same below.